components/config/basedir.rs

```diff
@@ -17,19 +17,19 @@ use std::path::PathBuf;
 use xdg;
 
 #[cfg(all(unix, not(target_os = "macos"), not(target_os = "ios"), not(target_os = "android")))]
-pub fn default_config_dir() -> Option<PathBuf> {
+pub fn default_config_dir() -> PathBuf {
     let xdg_dirs = xdg::BaseDirectories::with_profile("servo", "default").unwrap();
     let config_dir = xdg_dirs.get_config_home();
-    Some(config_dir)
+    config_dir
 }
 
 #[cfg(target_os = "android")]
 #[allow(unsafe_code)]
-pub fn default_config_dir() -> Option<PathBuf> {
+pub fn default_config_dir() -> PathBuf {
```

Comment:
There's a windows-specific version further down in the file as well.

Reply:
Noticed it. Thank you. Testing the fix locally now. Don't know how I missed that :D I will amend my commit soon.